hedera-node/src/main/java/com/hedera/services/utils/UnzipUtility.java

```diff
@@ -31,7 +31,7 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
 
-public class UnzipUtility {
+public final class UnzipUtility {
 
```

Comment:
SonarCloud also complains about a code smell of an implicit public constructor. We could add a private constructor to hide this implicit public one and just do a throw there
This is for your reference
https://androidhiro.com/source/android/example/java-private-constructor-checker/1955

Reply:
Added the default private constructor. None of the approaches mentioned in the reference works. I don't think we  want to introduce the package proposed in this article to have `PrivateConstructorChecker` for the test coverage. 

I will dig to see if there is other way to do it.